<html><h1>Validation FAILED.</h1><p>Build Time: 23 minutes 32 seconds 710 ms</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/0165438543642d47e8f17b7d8b1551918de1f422">0165438543642d47e8f17b7d8b1551918de1f422</a></p><h6>Task Summary:</h6><ul><li>subrepository-compile-jdk8 -  :heavy_check_mark:</li><li>subrepository-semantic-versioning-jdk8 -  :heavy_check_mark:</li><li>subrepository-pmd-jdk8 -  :heavy_check_mark:</li><li>subrepository-source-format-jdk8 -  :x:<pre><code>     [echo] java.lang.Exception: Found 2 formatting issues:
     [echo] 1: Missing parentheses, see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/documentation/parentheses.markdown: ./commerce-product-definitions-web/src/main/resources/META-INF/resources/attachment_file_entries.jsp 170 (SourceCheck:JSPParenthesesCheck)
     [echo] 2: Missing parentheses, see https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/documentation/parentheses.markdown: ./commerce-product-definitions-web/src/main/resources/META-INF/resources/attachment_file_entry_columns.jspf 18 (SourceCheck:JSPParenthesesCheck)
     [echo] 
     [echo] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:426)
     [echo] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:254)
     [echo] 
     [echo] FAILURE: Build failed with an exception.
     [echo] 
     [echo] * What went wrong:
     [echo] Execution failed for task ':checkSourceFormatting'.
     [echo] &gt; Process 'command '/opt/java/jdk1.8.0_121-x64/bin/java'' finished with non-zero exit value 1
     [echo] 
     [echo] * Try:
     [echo] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [echo] 
     [echo] * Exception is:
     [echo] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkSourceFormatting'.
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [echo] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [echo] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [echo] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [echo] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)</code></pre></li><li>central-requirements-jdk8/subrepository-unit-jdk8 -  :heavy_check_mark:</li></ul><h5>For full console, click <a href="https://test-1-5.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/82//consoleText">here</a>.</h5><hr/><div><h6>Test Results:</h6><div><div><p>46 Tests Passed.<br/>0 Tests Failed.</p></div></div><h5>For all test results, click <a href="https://test-1-5.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/82//testReport">here</a>.</h5></div></html>

--- a/commerce-product-definitions-web/src/main/resources/META-INF/resources/attachment_file_entries.jsp
+++ b/commerce-product-definitions-web/src/main/resources/META-INF/resources/attachment_file_entries.jsp
@@ -167,7 +167,7 @@ PortletURL portletURL = cpAttachmentFileEntriesDisplayContext.getPortletURL();
 
 										<liferay-ui:search-container-column-text>
 											<c:choose>
-												<c:when test="<%= Validator.isNull(thumbnailSrc) %>">
+												<c:when test="<%= Validator.isNull(thumbnailSrc) || (cpAttachmentFileEntry.getType() != 0) %>">
 													<liferay-frontend:icon-vertical-card
 														actionJsp="/attachment_file_entry_action.jsp"
 														actionJspServletContext="<%= application %>"

--- a/commerce-product-definitions-web/src/main/resources/META-INF/resources/attachment_file_entry_action.jsp
+++ b/commerce-product-definitions-web/src/main/resources/META-INF/resources/attachment_file_entry_action.jsp
@@ -40,6 +40,7 @@ else {
 		<portlet:param name="mvcRenderCommandName" value="editCPAttachmentFileEntry" />
 		<portlet:param name="cpDefinitionId" value="<%= String.valueOf(cpAttachmentFileEntry.getClassPK()) %>" />
 		<portlet:param name="cpAttachmentFileEntryId" value="<%= String.valueOf(cpAttachmentFileEntry.getCPAttachmentFileEntryId()) %>" />
+		<portlet:param name="type" value="<%= String.valueOf(cpAttachmentFileEntry.getType()) %>" />
 	</portlet:renderURL>
 
 	<liferay-ui:icon

--- a/commerce-product-definitions-web/src/main/resources/META-INF/resources/attachment_file_entry_columns.jspf
+++ b/commerce-product-definitions-web/src/main/resources/META-INF/resources/attachment_file_entry_columns.jspf
@@ -15,19 +15,13 @@
 --%>
 
 <c:choose>
-	<c:when test="<%= Validator.isNotNull(thumbnailSrc) %>">
+	<c:when test="<%= Validator.isNotNull(thumbnailSrc) && (type == 0) %>">
 		<liferay-ui:search-container-column-image
 			name="image"
 			src="<%= thumbnailSrc %>"
 			toggleRowChecker="<%= true %>"
 		/>
 	</c:when>
-	<c:otherwise>
-		<liferay-ui:search-container-column-icon
-			icon="documents-and-media"
-			toggleRowChecker="<%= true %>"
-		/>
-	</c:otherwise>
 </c:choose>
 
 <liferay-ui:search-container-column-text

